app/controller/dashboard.js

```diff
@@ -1,75 +1,63 @@
+// Imports for packages
 import Hubkit from 'hubkit';
 import Promise from 'bluebird';
 import _ from 'lodash';
 import ini from 'ini';
 
+// Local Imports
 import app from '~/';
 import { isBeta } from './auth';
 import { Application } from '~/model/application';
 
-// Create Hubkit with a static instance
-let gh = new Hubkit();
```

Comment:
your comment about the promisify and map/filter was a good point — I forgot that we're polyfilling the Promise object with Bluebird's extra functions. That's a perfectly fine thing to do, then, if you do actually need those extra functions. I'd just maybe add a comment like "polyfill bluebird methods" or something like that so that it's clear why you're re-promisifying an already promise-based library :)